src/Docs/manual.tex

```diff
@@ -5697,71 +5698,72 @@ \subsection{Argument description for the control input file ``tcntl''}
 %----------------------------------------------------------------------
 \brules{optional} 
 \bdescr{writes a input file for OPENDX\cite{opendx} to show the moving atoms as a
-  ball-stick model. Bonds are drawn for every pair of atoms that are
-  closer than 1.2 times the sum of the covalent radii.}
+	ball-stick model. Bonds are drawn for every pair of atoms that are
```

Comment:
Find out the difference. By all means avoid non-printing character such as tabs

Reply:
solved with commit [c3ba0c1](https://github.com/cp-paw/cp-paw/pull/8/commits/c3ba0c18001e93403aa848bbfbe1336fdd54c362)